packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -117,18 +117,58 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  // The ButtonBar widget is planned to be deprecated in favor of the OverflowBar
+  // widget. This ButtonBar implementation uses OverflowBar widget internally
+  // for backward compatibility.
   'ButtonBar': (BuildContext context, DataSource source) {
-    // not implemented: buttonTextTheme
-    return ButtonBar(
+    final EdgeInsetsGeometry buttonPadding = ArgumentDecoders.edgeInsets(source, ['buttonPadding']) ?? const EdgeInsets.all(8.0);
+    final ButtonBarLayoutBehavior layoutBehavior = ArgumentDecoders.enumValue<ButtonBarLayoutBehavior>(ButtonBarLayoutBehavior.values, source, ['layoutBehavior'])
+      ?? ButtonBarLayoutBehavior.padded;
+
+    Widget overflowBar = OverflowBar(
       alignment: ArgumentDecoders.enumValue<MainAxisAlignment>(MainAxisAlignment.values, source, ['alignment']) ?? MainAxisAlignment.start,
-      mainAxisSize: ArgumentDecoders.enumValue<MainAxisSize>(MainAxisSize.values, source, ['mainAxisSize']) ?? MainAxisSize.max,
-      buttonMinWidth: source.v<double>(['buttonMinWidth']),
```

Comment:
this removes support for `buttonMinWidth` which violates our promise of keeping 100% backwards compatibility.

Reply:
Apologies for not mentioning it in the description.

`buttonMinWidth` doesn't do anything since the implementation of new buttons based on `ButtonStyleButton`.   Please read Hans' comment [here](https://github.com/flutter/flutter/issues/79171#issuecomment-819695638) for more details.

The older Material buttons that used this property are no available in the framework. When this parameter was implemented, those older Material buttons used the provided value from `buttonMinWidth` parameter to adjust their min width.

Reply:
let's at a minimum add some documentation to the big comment at the top of this function so that the API docs acknowledge that the ButtonBar widget has changed over time.

Reply:
`buttonMinWidth`, `buttonHeight`, and `buttonAlignedDropdown` are no longer supported by widgets in the framework. I added some documentation in the comment above.

Reply:
That comment is good. I was thinking more of the API documentation around line 52, though (maybe a new paragraph that says something about like "Some features have changed in the underlying Material library and are therefore no longer supported, including:" followed by a description of what's happened with `ButtonBar`).

Reply:
Sounds good.